index.html

```diff
@@ -740,6 +770,59 @@
     margin-top: 8px;
     margin-bottom: 8px;
   }
+  .flex {
+    display: flex;
+  }
+  .toolbarsearchbox {
+    align-items: center;
+  }
+  .searchbarcollapse {
+    align-items: center;
+    height: 100%;
+    background-color: white;
+    padding-left: 8px;
+    padding-right: 8px;
+  }
+  .popupcontainerholder {
+    width:100%;
+  }
+  .rightfloat {
+    float: right;
+  }
+  .headerbarpage {
+    width: 100%;
+    display: flex;
+    align-items: center;
+  }
+  .headerbarpages {
+    width: 100%;
+  }
+  .headerbar {
+    padding: 0px;
+  }
+  .searchbackground {
+    background-color: white;
+  }
+  .grey {
+    color: gray;
+  }
+  paper-typeahead-input {
+    display: inline-flex;
+    width: calc(100% - 60px);
+  }
+  .headercenteredicon {
```

Comment:
Yes, that is necessary. The back `paper-icon-button` on my search bar page needs it to work.